format-delimited/src/main/java/io/cdap/plugin/format/delimited/input/DelimitedConfig.java

```diff
@@ -77,6 +77,13 @@ public long getSampleSize() {
     return Long.parseLong(getProperties().getProperties().getOrDefault(NAME_SAMPLE_SIZE, "1000"));
   }
 
+  public DelimitedConfig() {
+    super();
+  }
```

Comment:
should be a newline after between methods

Reply:
Done